js/components/tabs.js

```diff
@@ -75,33 +79,52 @@ var Tabs = React.createClass({
         var ref = ("tab-menu-" + (index + 1));
         var title = panel.props.title;
         var activeTabStyle = {};
+        var isSelected = this.state.tabActive === index + 1;
```

Comment:
It looks like we have multiple usages of index + 1 in this function. Can we store this in a var?

Reply:
I'll update it